src/debug/di/shimdatatarget.h

```diff
@@ -85,6 +82,13 @@ class ShimDataTarget : public ICorDebugMutableDataTarget
 
     // @dbgtodo - add Native Patch Table support
 
+    //
+    // ICorDebugDataTarget4
+    //    
+
+    // Get the OS Process ID that this DataTarget is for.
+    virtual DWORD STDMETHODCALLTYPE GetPid();    
```

Comment:
Lets make this virtual HRESULT STDMETHODCALLTYPE GetPid(DWORD* pPid);

We want to be able to return errors like not implemented or not available without any confusion if a process ever has the same PID as one of those error constants.